src/app/shared/components/cluster-summary/style.scss

```diff
@@ -69,3 +69,12 @@
     margin-left: -15px;
   }
 }
+
+.km-event-rate-limit {
```

Comment:
As mentioned `km-` prefix is for the globally available styles only. Here is it a local style.

s/km-event-rate-limit/event-rate-limit